cpp/proto/arcticc/pb2/descriptors.proto

```diff
@@ -163,6 +163,8 @@ message NormalizationMetadata {
 
         // Pandas series support arbitrary objects but restricting to a string seems reasonable
         string name = 1;
+        // Backwards-compatible way of treating name as optional
+        bool has_name = 9;
```

Comment:
Worth a comment below `columns = 8` to explain that there is a field with index `9`